styles/youtube/catppuccin.user.css

```diff
@@ -973,6 +962,23 @@
       .yt-spec-touch-feedback-shape__fill {
       background-color: @text;
     }
+        /* Transcript Button */
```

Comment:
Doesn't this fix comment/reply buttons? Why is this saying they are transcript stuff?